lib/getComment.js

```diff
@@ -1,17 +1,13 @@
-module.exports = getComment
-
 function getComment (comment, defaultComment) {
-  // comment can be either a string or an array
-
-  if (!comment) {
-    comment = defaultComment
-  }
+  // set comment to default if none passed it
+  const eventComment = (!comment) ? defaultComment : comment
```

Comment:
I think this can just be `comment || defaultComment`.

Reply:
oh right i always forget you can do that, thanks!